pym/bob/input.py

```diff
@@ -3447,15 +3478,15 @@ def __parse(self, envOverrides, platform, recipesRoot=""):
             self.__parseUserConfig(c)
 
         # calculate start environment
-        osEnv = Env(os.environ)
-        osEnv.setFuns(self.__stringFunctions)
```

Comment:
That line should be retained. The layers can add something to `self.__stringFunctions` and that *could* be lost. I don't think it is (because we pass a dict and store it) but it is misleading, nonetheless.